test/runtests.jl

```diff
@@ -261,4 +261,14 @@ end;
     end
     @test sum(res) >= N*0.80
 end
-    
\ No newline at end of file
+
+@testset "SimpleHypergraphs connected components" begin
+    bip = LightGraphs.SimpleGraph(BipartiteView(h1))
+    cc = LightGraphs.connected_components(bip)
+    filter!.(<=(size(h1, 1)), cc)
+    filter!(!isempty, cc)
+
+    cc2 = SimpleHypergraphs.get_connected_components(h1)
+    @test sort!(sort!.(cc)) == sort!(sort!.(cc2))
+    typeof(cc2) == Array{Array{Int64,1},1}
```

Comment:
shorter is:
`Vector{Vector{Int}}`
note that normally we use os platform-specific  `Int` rather than `Int64`

Additionally, there is no `@test` in from of this line. So it is not doing anything.